src/mock-data/currentUser.json

```diff
@@ -0,0 +1,5 @@
+{
+    "id": "ewwboc7m",
```

Comment:
We've moved mock-data out of src and into the top level directory as part of configurable data